scripts/ci/package.sh

```diff
@@ -5,4 +5,12 @@
 set -euxo pipefail
 
 cd build
-cpack -C "${AUDACITY_BUILD_TYPE}" --verbose
+
+if [[ "${OSTYPE}" == msys* && ${GIT_BRANCH} == release* ]]; then # Windows
+    cmake --build . --target innosetup --config "${AUDACITY_BUILD_TYPE}"
```

Comment:
Any reason for making `innosetup` a target rather than a CPack pre/post build script or external generator? If no, we can merge as-is and change it later.